code/src/App.js

```diff
@@ -1,9 +1,130 @@
-import React from 'react'
+import React, { useState } from 'react'
+import './App.css'
+
+
+
+const ageGroups = ["young", "old", "perfect"]
 
 export const App = () => {
+  const [ageGroup, setAgeGroup] = useState("perfect")
+  const [animal, setAnimal] = useState("")
+  const [showResult, setShowResult] = useState(false)
+  const [bestFriend, setBestFriend] = useState("")
+  const [fun, setFun] = useState("fun")
```

Comment:
It looks like this can be a bool?
```suggestion
  const [fun, setFun] = useState(true)
```